README.md

```diff
@@ -53,6 +53,8 @@ Currently the following services are supported:
   - Apple Music
   - Tidal
 
+Note that bot doesn't react to YouTube links in group chats. Reasons are [here](9dogs/tg-odesli-bot/blob/master/README.md).
```

Comment:
The link is broken. And where it should lead?

Reply:
I think we can link this issue: https://github.com/9dogs/tg-odesli-bot/issues/12

Reply:
Yeah sorry, didn't spot that I pasted the wrong link. It was supposed to lead to the issue mentioned